.github/workflows/on-tag.yml

```diff
@@ -0,0 +1,23 @@
+name: Release charts, docker images and binaries
+
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+      - "[0-9]+.[0-9]+.[0-9]+*"
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    uses: ./.github/workflows/reusable/build-and-test.yml
+    secrets: inherit
+  docker-hub-image:
+    needs: [test]
+    uses: ./.github/workflows/reusable/push-docker-image.yml
+    secrets: inherit
+    with:
+      image_name: scalingln/simln
```

Comment:
Can be updated to [bitcoindevproject](https://hub.docker.com/u/bitcoindevproject) 👍

Reply:
Updated